openmrs-elis-atomfeed-client-omod/pom.xml

```diff
@@ -196,6 +196,7 @@
             <groupId>org.openmrs.api</groupId>
             <artifactId>openmrs-api</artifactId>
             <version>${openMRSVersion}</version>
+            <type>jar</type>
```

Comment:
is this required? by default this would be jar, right?

Reply:
Resolved